### PR TITLE
bash-completion@2 2.12.0

### PR DESCRIPTION
**Pull request by moonfruit**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by bevanjkay — APPROVED** [review]

**Comment by cho-m** [comment]
Need to remove conditional. Should only need the `build.head?` logic.
https://github.com/Homebrew/homebrew-core/blob/85e3a89931abfeee637c2cfec5a462cb09a88d64/Formula/b/bash-completion%402.rb#L39-L47

File: Formula/b/bash-completion@2.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class BashCompletionAT2 < Formula
   desc "Programmable completion for Bash 4.2+"
   homepage "https://github.com/scop/bash-completion"
-  url "https://github.com/scop/bash-completion/releases/download/2.11/bash-completion-2.11.tar.xz"
-  sha256 "73a8894bad94dee83ab468fa09f628daffd567e8bef1a24277f1e9a0daf911ac"
+  url "https://github.com/scop/bash-completion/releases/download/2.12.0/bash-completion-2.12.0.tar.xz"
+  sha256 "3eb05b1783c339ef59ed576afb0f678fa4ef49a6de8a696397df3148f8345af9"
   license "GPL-2.0"
```
Comment (p-linnane) [inline Formula/b/bash-completion@2.rb:6]:
```suggestion
  license "GPL-2.0-or-later"
```
Ref: license headers in the completion file.

**Comment by moonfruit** [comment]
> Need to remove conditional. Should only need the `build.head?` logic.
> 
> https://github.com/Homebrew/homebrew-core/blob/85e3a89931abfeee637c2cfec5a462cb09a88d64/Formula/b/bash-completion%402.rb#L39-L47

Done.

**Comment by p-linnane** [comment]
Thanks! Can you squash your commits into one with the message `bash-completion@2 2.12.0`? Then we can get this merged.

**Review by p-linnane — APPROVED** [review]

**Pull request merged**